README.md

```diff
@@ -1,24 +1,163 @@
-# Sensu Go Remediation Handler
+# Sensu Remediation Handler
 
-The Sensu Go Remediation Handler is a simple [Sensu Go event handler][handler]
-for configuring self-healing workflows using Sensu. This handler was heavily
-inspired by [Nick Stielau's "Sensu Remediator"][remediator] (circa 2012).
+[![Bonsai Asset Badge](https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-remediation-handler) [![Build Status](https://travis-ci.com/sensu/sensu-remediation-handler.svg?branch=master)](https://travis-ci.com/sensu/sensu-remediation-handler)
 
-The Sensu Go Remediation Handler &ndash; and other similar "self healing"
-workflows in Sensu &ndash; use a few simple Sensu features:
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Environment Variables](#environment-variables)
+  - [Annotations](#annotations)
+    - [Annotation Specification](#annotation-specification)
+    - [Remediation Action Specification](#remediation-action-specification)
+- [Setup](#setup)
+- [Examples](#examples)
+  - [Example "Unscheduled" Check (Remediation Action)](#example-unscheduled-check-remediation-action)
+  - [Example Check Definition and Remediation Request Configuration](#example-check-definition-and-remediation-request-configuration)
+- [Acknowledgements](#acknowledgements)
 
-- An "unscheduled check" configuration (i.e. a Sensu Check with the `"publish":
+## Overview
+
+The Sensu Remediation Handler is a [Sensu Go event handler][handler]
+that enables you to build self-healing workflows in Sensu. 
+
+The Sensu Remediation Handler &ndash; and other similar self-healing
+workflows in Sensu &ndash; combine a few Sensu features:
+
+- An "unscheduled check" configuration (i.e. a Sensu check with the `"publish":
   false` attribute set).
-- The Sensu Checks API `POST /checks/:check/execute` endpoint, which provides
-  the ability to make ad hoc check requests.
-- The Sensu Agent's built-in entity subscriptions (e.g. `entity:web-server-01`)
+- The Sensu agent's built-in entity subscriptions (e.g. `entity:web-server-01`),
   which make it possible to target a single agent with a check execution
-  request.  
+  request.
+- The Sensu Checks API `POST /checks/:check/execute` endpoint, which allows
+ issuing ad hoc check requests.
```

Comment:
```suggestion
 you to issue ad hoc check requests.
```

Reply:
I think "issuing" doesn't make it clear who/what is doing the issuing. But maybe the user isn't either?

Reply:
In the case of using this handler, the handler is using this API to issue check requests based on defined remediation actions.